metadata.json

```diff
@@ -36,7 +36,6 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
         "8",
```

Comment:
There is no more EL-compatible OS left that you test on "EL8", so it's hard to claim support for RHEL8?